start-wizard.sh

```
@@ -0,0 +1,134 @@
+if [ "$confirm" != "Y" ] && [ "$confirm" != "y" ]; then
+    printf "\n  ${YELLOW}Aborted.${NC}\n\n"
+    exit 0
+fi
+
+# ── start ─────────────────────────────────────────────────────────────────────
+printf "\n  Starting XDCStats (%s)…\n\n" "$ENV_NAME"
+
+docker compose -f "$REPO_ROOT/docker-compose.yml" up -d
+
+printf "\n"
+printf "  ${GREEN}${BOLD}Started!${NC}\n"
```

Shellcheck (0.11.0)

[info] Don't use variables in the printf format string. Use printf '..%s..' "$foo".

(SC2059)